src/style.css

```diff
@@ -0,0 +1,99 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.search-form {
+  position: relative;
+  margin: 20px auto 0;
+}
+
+.img {
+  position: absolute;
+}
+
+.img-big {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.img-small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-form__input {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  font-weight: 300;
+  font-family: inherit;
+}
+
+.search-form__input-big {
+  height: 70px;
+  padding-left: 62px;
+
+  font-size: 16px;
+  line-height: 22px;
+
+  color: #3d4e61;
+}
+
+.search-form__input-small {
+  height: 42px;
+  padding-left: 33px;
+
+  font-size: 14px;
+  line-height: 19px;
+
+  color: #3d4e61;
+}
+
+.search-form__input::placeholder {
+  color: inherit;
+  font-weight: 300;
+}
+
+.search-form__input-big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-form__input-small::placeholder {
+  font-size: 14px;
+  line-height: 22px;
+}
+
+.search-form__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-form__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 600;
```

Comment:
U doesn't have font-weight: 600; Because u import only 300 and 800
<img width="246" alt="image" src="https://user-images.githubusercontent.com/86183007/185074095-4b5f6b59-2c3b-4add-8132-57300aeeddb5.png">